src/Contexts/MountedComponentsContext.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component, createContext } from 'react';
+
+export const MountedComponentsContext = createContext();
+
+class MountedComponentsContextProvider extends Component {
+  state = {
+    mountedComponentsDict: {
+      primaries: {
+        CraftingWindow: false,
+        ItemInfiWindow: false,
+        EnergyPurchaseWindow: false,
+        CoinsPurchaseWindow: false,
+      },
+      secondaries: {
+        CoinsTab: false,
+        EnergyTab: false,
+      },
+    },
+  };
+
+  mountOrunmountComponents = (componentsToMountArray, shouldMount) => {
```

Comment:
- update accordingly to removal of categories
 - please add a check for componentsToMountArray to actually be an array